qannotate/src/au/edu/qimr/qannotate/utils/SampleColumn.java

```diff
@@ -10,7 +10,6 @@
 import org.qcmg.common.vcf.header.VcfHeaderRecord;
 import org.qcmg.common.vcf.header.VcfHeaderUtils;
 
-@Deprecated
 public class SampleColumn {
```

Comment:
This should still be deprecated - want to encourage people to use `VcfFileMeta` instead. Its mostly test and unused classes (`VcfToMafTmp` and `CustomerConfidenceMode`) that use this. Maybe add a comment to the deprecation annotation telling users to use `VcfFileMeta` instead?

Reply:
Thanks for explaiantaion, I put the "Deprecated" back, and will add your suggested comment on next pull request